### ISSUE TITLE
 loss = nan

### ISSUE THREAD

**Issue by lixyresearch** [BVLC/caffe#342]
Check failed: (cudaMalloc(&gpu_ptr_, size_)) == cudaSuccess (2 vs. 0) 
I set the batchsize=128
21 19:47:57.527614  3132 solver.cpp:208] Iteration 240, lr = 0.01
I0421 19:47:57.527961  3132 solver.cpp:65] Iteration 240, loss = 6.55221
I0421 19:48:34.318344  3132 solver.cpp:208] Iteration 260, lr = 0.01
I0421 19:48:34.318675  3132 solver.cpp:65] Iteration 260, loss = 6.55303
I0421 19:49:10.907234  3132 solver.cpp:208] Iteration 280, lr = 0.01
I0421 19:49:10.907579  3132 solver.cpp:65] Iteration 280, loss = nan
I0421 19:49:47.561594  3132 solver.cpp:208] Iteration 300, lr = 0.01
I0421 19:49:47.561944  3132 solver.cpp:65] Iteration 300, loss = nan
